lib/orderbook/OrderBookRepository.ts

```diff
@@ -1,104 +1,18 @@
-import { default as Sequelize, Op } from 'sequelize';
-
-import { db, orders } from '../types';
-import Logger from '../Logger';
-import DB, { Models } from '../db/DB';
 import Bluebird from 'bluebird';
-import { OrderAttributes } from '../types/db';
-
-type Orders = {
-  buyOrders: db.OrderInstance[];
-  sellOrders: db.OrderInstance[];
-};
+import { db } from '../types';
+import DB, { Models } from '../db/DB';
 
 class OrderbookRepository {
```

Comment:
Now that this class isn't actually fetching orders from a db repository any longer, it seems like it's misnamed. In fact, I think we can probably get rid of this class altogether. I figure we can extract the remaining logic in this class (which is 3 lines, really) and move it directly to `Orderbook.ts`.

Reply:
i'd suggest to keep all `sequelize` object wrapped separately, and keep this class for now (might get more traction later). I think it's reasonable that currency/pairs management is under order book domain. If not, we can open a generic repository. I think it's good to keep it separate from `DB` or service classes.